packages/react/src/components/UIShell/components/SideNavMenu.tsx

```diff
@@ -172,25 +172,6 @@ export const SideNavMenu = React.forwardRef<HTMLElement, SideNavMenuProps>(
 
       // if depth is more than 0, that means its nested, thus we set treeview mode
       setIsTreeview?.(true);
-
-      if (isTreeview) {
-        const calcButtonOffset = () => {
-          // menu with icon
-          if (children && IconElement) {
-            return depth + 3;
-          }
-
-          // menu without icon
-          if (children) {
-            return depth * 4;
-          }
-          return depth;
-        };
-
-        if (buttonRef.current) {
-          buttonRef.current.style.paddingLeft = `${calcButtonOffset()}rem`;
-        }
-      }
```

Comment:
To give a little background on these JS styles, it's how the Treeview component sets the spacing programmatically if the nesting indefinitely continues. I added that here in case we needed to account for X amount of nesting levels, but design-wise it seems like we're only sticking to set amount of them, so this refactor makes sense! 👍